aspnet/dnx/projects.rst

```diff
@@ -1,8 +1,652 @@
\ No newline at end of file
+Creating a new project 
+----------------------
+
+At its simplest form, a DNX project is a **project.json** file along with a code file:
+
+::
+
+    -MyApp (folder)
+    --project.json
+    --Program.cs
+
+There are two mandatory conditions for a project. The *project.json* file must contain valid json, where brackets *{ }* are used inside the file, and that your *program.cs* file contains valid C#. 
+
+The presence of a *project.json* file defines a DNX project. It is the *project.json* file that contains all the information that the DNX needs to run and package your project. For additional details, including the *project.json* file schema, see `Project.json <https://github.com/aspnet/Home/wiki/Project.json-file>`_.
+
+When using some editors there are other files that you'll see. For instance, a file with the extension .xproj is used in the case of Visual Studio. These other types of files are requirements of their specific tool, but are not a requirement of the DNX. The .xproj file type, for example, is an MSBuild file that is used by Visual Studio and keeps information that is important to Visual Studio, but this type of file does not impact the DNX.
+
+Targeting frameworks
+--------------------
+
+One of the sections you can add to the *project.json* is the ``frameworks`` section. The ``frameworks`` section looks like this:
+
+.. code-block:: json
+
+    {
+      "frameworks": {
+        "dnx451": {},
+        "dnxcore50": {}
+      }
+    }
+
+Each of the values in the ``frameworks`` section of the *project.json* file is a framework that your project will compile. The above snippet will build for the full .NET Framework (dnx451) or Core CLR (dnxcore50). 
+
+If you pack this project, using ``dnu pack`` then you will get a NuGet package that looks like the following:
+
+.. image:: projects/_static/TargetPlatformNupkg.png
+
+Notice that the ``nupkg`` has a folder for each of the frameworks you specified, allowing this NuGet package to work on any of the frameworks. You can also specify this as follows:
+
+.. code-block:: json
+
+    {
+      "frameworks": {
+        "dnxcore50":{},
+        ".NETPortable,Version=v4.5,Profile=Profile7":{}
+      }
+    }
+
+With the above ``frameworks`` section, you'll generate a package with a *portable-net45+win* folder that will be used when running on platforms that match that portable profile.
+
+Adding dependencies
+-------------------
+
+You manage the dependencies of your application with the ``dependencies`` section of your *project.json* file. The dependencies are defined by name and version, where the runtime loaders determine what should be loaded.
+
+.. code-block:: json
+
+    {
+      "dependencies":{
+        "Microsoft.AspNet.Mvc": "6.0.0-beta4"
+      }
+    }
+
+The above *project.json* section tells the DNX that you depend upon ``Microsoft.AspNet.Mvc``, which means you also depend on everything that the MVC package depends on.
```

Comment:
You depend on a name and a version, it isn't necessarily a nuget package. You depend on everything that MVC depends on. But it could be the source of MVC or a package.